.github/workflows/main.yml

```diff
@@ -90,19 +90,30 @@ jobs:
           fi
 
   integration-test:
+    name: integration-test (${{ matrix.task.name }}, ${{ matrix.task.installer }}, ${{ matrix.pyodide-version }}-pyodide, ${{ matrix.os }})
     runs-on: ${{ matrix.os }}
     needs: [check-integration-test-trigger]
     strategy:
       fail-fast: false
       matrix:
         task: [
-          {name: test-recipe, installer: pip},
-          {name: test-src, installer: pip},
-          {name: test-recipe, installer: uv},
-          {name: test-src, installer: uv},
-          {name: test-integration-marker, installer: pip},  # installer doesn't matter
-        ]
+            { name: test-recipe, installer: pip },
+            { name: test-src, installer: pip },
+            { name: test-recipe, installer: uv },
+            { name: test-src, installer: uv },
+            { name: test-integration-marker, installer: pip }, # installer doesn't matter
+          ]
         os: [ubuntu-latest, macos-latest]
+        pyodide-version: [stable]
```

Comment:
(no action required for this PR)

For a reproducible test, maybe we should specify the pyodide version tested in the integration test somewhere, instead of letting pyodide-build grab the last compatible version automatically